torch_geometric/loader/link_neighbor_loader.py

```diff
@@ -79,16 +101,43 @@ def _create_label(self, edge_label_index, edge_label):
 
         edge_label = torch.cat([pos_edge_label, neg_edge_label], dim=0)
 
-        return edge_label_index, edge_label
+        return edge_label_index, edge_label, edge_time
+
+    def _modify_node_time(self, edge_label_index, edge_time):
+        """For edges in a batch replace `src` and `dst`
```

Comment:
Hmm curious if this makes sense? Doesn't it mean that any node that is used in a future edge cannot be sampled? I.e if a node has an old and newer edge, we cannot sample the old edge?

Reply:
Sorry I don't get your question fully. 
`I.e if a node has an old and newer edge, we cannot sample the old edge`
The temporal sampler only makes sure nodes obey temporal constraints, as in we don't check if the edge obeys the constraint.

Reply:
Okay right. Do you think we should consider adding support for this in the temporal sampler directly?

Reply:
Yeah in the long term we should we should support any edge in the graph with a timestamp or `edge_label_time`  directly in the `torch_sprase` sampler.